src/java.base/windows/native/libjli/java_md.c

```diff
@@ -321,6 +323,8 @@ GetJREPath(char *path, jint pathsize)
         }
     }
 
+    JLI_TraceLauncher("GetJREPath - attempt to get JRE location from shared lib of the image\n");
```

Comment:
Maybe add a trace also in the USE_REGISTRY_LOOKUP section

Reply:
> I am not sure if this even works any more. Maybe Alan could comment on this ?

The GetPublicJREHome function was removed at some point, I think JDK 9, as it didn't make sense to have in the OpenJDK project. However, Oracle installer did depend on it and will require a bit of effort to see if removing it now will cause an issue now or not.

Reply:
@MBaesken I checked into the building with -DUSE_REGISTRY_LOOKUP as that compiled in code that the Oracle installer needed when it copied java.exe to somewhere. This is indeed left over code and can be removed from java_md.c.

Reply:
Hi Alan, thanks for checking this !
I removed the USE_REGISTRY_LOOKUP usages.